content/en/xlt/load-testing/Manual/570-scorecard.md

```diff
@@ -9,7 +9,210 @@ description: >
+        "description": "Just any text that explains the purpose", //  the text to display in the report, kinda help text like - optional, default empty
+        // the list of rules assigned to this group and to evaluate in that order - required, at least one rule per group
+        "rules": ["homepageResponseTimeA", "homepageResponseTimeB", "homepageResponseTimeC"]
+    }
+]
+```
+
+Groups are evaluated in the order of definition. Each rule evaluation triggers the evaluation of all rules assigned to it, in order of assignment. Rules that do not pass won't stop the evaluation of the remaining rules.
+
+The number of achievable points as well as the computed number of achieved points depend on the group's **mode**:
+
+| Mode | Achievable Points | Achieved Points |
+|--------|--------|--------|
+| _firstPassed_ | Max. of all rules' achievable points | First rule that passes |
+| _lastPassed_ | Max. of all rules' achievable points | Last rule that passes  |
+| _allPassed_ | Sum of all rules' achievable points | Sum of all rules that pass  |
+
+
+### Rating
+
+You can define a final rating for the entire test including the ability to mark it failed, following this pattern:
+
+```json
+{
+    "id": "poor", // the ID of the rating for reference and listing in the report (used as fallback when no name is given) - required and unique
+    "name": "Poor", // the name of the rating for reference and listing in the report - optional
+    "enabled": true, // helps to disable the rating for testing purposes - optional, default = true
+    "comment": "Lorem ipsum ...", // some comment; won't be evaluated - optional
+    "description": "Load test performed poorly", //  the text to display in the report, kinda help text like - optional, default empty    
+    "value": 50.0, // the upper limit for the rating  - required, must be greater than or equal to 0.0 and lower than or equal to 100.0
+    "failsTest" : true // whether to mark the load test as failed when applied - optional, default = false
+} 
+```
+
+To determine the test rating, all points from all rule groups will be summed up to calculate a percentage. Depending on the overall percentage of points achieved, a final rating is chosen from the list of ratings specified by the user and applied to the test. The ratings are ordered by value and have to cover 0 to 100%. The list of ratings is processed in the same order as specified and the first applicable rating defines the final rating. A rating is applicable when its value is greater than or equal to the overall percentage of achieved points.
+
+So if we got a total of 30 achievable points and the passing rules evaluate to a total of 20, we have 66.7% as a final result, which would amount to the rating "Ok" in the example below: 
+
+```json
```

Comment:
JSON does not support comments. Either we have to use js as language for highlighting or remove the comments.

```suggestion
```js
```